<?xml version="1.0"?>
<launch>


<node pkg="planner_utils" type="costmap_cloner_node" name="costmap_cloner_node" output="screen">

<param name="map_cloner/publish_frequency" value="10.0"/>
<param name="map_cloner/width" value="50.0"/>
<param name="map_cloner/height" value="50.0"/>
<param name="map_cloner/resolution" value="0.1"/>

<param name="map_cloner/global_frame" value="/odom"/>
<param name="map_cloner/robot_base_frame" value="/base_link"/>
<param name="map_cloner/rolling_window" value="false"/>

<rosparam param="map_cloner/plugins">
[{'name':'static','type':'planner_utils::CloneLayer'}]
</rosparam>


<param name="map_cloner/static/map_topic"
value="/rcv_move_base/move_base/local_costmap/costmap"/>

<param name="map_cloner/static/subscribe_to_updates" value="true"/>

<param name="map_cloner/static/lethal_cost_threshold" value="254"/>
<param name="map_cloner/static/unknown_cost_value" value="255"/>
<param name="map_cloner/static/trinary_costmap" value="false"/>
<param name="map_cloner/static/use_maximum" value="true"/>


<param name="map_cloner/inflation/inflation_radius" value="2.0" />
<param name="map_cloner/inflation/cost_scaling_factor" value="1.0" />

<param name="map_cloner/footprint_stamped_topic"
value="/rcv_move_base/move_base/local_costmap/footprint" />

<param name="map_cloner/footprint_topic"
value="/map_cloner/footprint_to_costmap_input" />

</node>


</launch>
